pom.xml

```diff
@@ -113,6 +113,7 @@
         <module>org.eclipse.winery.highlevelrestapi</module>
         <module>org.eclipse.winery.tools.copybaragenerator</module>
         <module>org.eclipse.winery.model.substitution</module>
+        <module>org.eclipse.winery.crawler</module>
```

Comment:
could we have this added to the list of artifacts bundled by the p2-maven-plugin a few lines down? Thanks

Reply:
Progress on the PR integrating winery into container has obsoleted this change request